gini/gini-switch-sdk-ios
# Simulator default image

# Introduction

The camera is not available on the Simulator. In that case, we inject a default image when the user taps on the capture button so that the app can still be tested.

# How to test

Start the sample app in the Simulator, take a picture and see if the default image shows as if it was taken by the camera.

# Merge info

Automatic
